### DISCUSSION TITLE
PaddleOCR/ppstructure/blog/how_to_do_kie

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#13790] [Q&A]
# PaddleOCR/ppstructure/blog/how_to_do_kie

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/ppstructure/blog/how_to_do_kie.html

**Comment by DingZhaohai** [comment]
可以
